scripts/examples/esp_example.sh

```diff
@@ -29,18 +30,26 @@ if [ -z "$app" ]; then
     exit 1
 fi
 
+git -C "$IDF_PATH" submodule update --init --recursive
+cd "$IDF_PATH"
+./install.sh
+. ./export.sh
+
+cd "$work_dir"
```

Comment:
Why was this not required previously and is required now? cc @andy31415

Reply:
@drishi I have changed the current directory to $IDF_PATH for running `install.sh` and `export.sh`, hence I have added that.

Reply:
Ok. But, IMO, install.sh was happening before this as well. We need to check where does that come from

Reply:
Yes @dhrishi right. I have used export.sh only, `install.sh` happens in Dockerfile.